charts/values.yaml

```diff
@@ -111,6 +111,10 @@ litmusPortalServer:
         tag: ci
```

Comment:
values.yaml needs to change as per latest changes upstream

Reply:
I took refernce from https://github.com/litmuschaos/litmus/blob/master/litmus-portal/k8s-manifest.yml#L64